cli/lib/kontena/cli/grids/common.rb

```diff
@@ -117,5 +117,68 @@ def get_grid(name = nil)
         exit_with_error "No grid given or selected"
       end
     end
+
+    module Parameters
+      def self.included(base)
+        base.class_eval do
+          option "--default-affinity", "[AFFINITY]", "Default affinity rule for the grid", multivalued: true
+          option "--no-default-affinity", :flag, "Unset grid default affinity"
+          option "--statsd-server", "STATSD_SERVER", "Statsd server address (host:port)"
+          option "--no-statsd-server", :flag, "Unset statsd server setting"
+          option "--log-forwarder", "LOG_FORWARDER", "Set grid wide log forwarder (set to 'none' to disable)"
+          option "--log-opt", "[LOG_OPT]", "Set log options (key=value)", multivalued: true
+        end
+      end
+
+      def validate_log_opts
+        if !log_opt_list.empty? && log_forwarder.nil?
+          raise Kontena::Errors::StandardError.new(1, "Need to specify --log-forwarder when using --log-opt")
+        end
+      end
+
+      def parse_log_opts
```

Comment:
Not exactly the same thing though, because this also collects those into a single `Hash`?